src/components/ResortsData.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import ResortsList from './ResortsList';
+import ResortsListHead from './ResortsListHead';
+import Pagination from './Pagination';
+
+class ResortsData extends Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      data: [],
+      page: 1,
+      isLoading: false,
+      error: null
+    }
+  }
+
+  componentDidMount(){
+    this.setState({isLoading: true})
+    fetch(`http://localhost:3000/resorts?_page=${this.state.page}`)
+        .then(response => {
+          if (response.ok) {
+            return response.json();
+          } else {
+            throw new Error('Something went wrong ...');
+          }
+        })
+        .then(data => this.setState({ data: data, isLoading: false }))
+        .catch(error => this.setState({ error, isLoading: false}))
+  }
+
+  handleClick(e) {
+    this.setState({page: (Number(e.target.id))});
```

Comment:
It's not fetching data, because it is triggered only in `componentDidMount` which will run only once on the beginning, not after state change.

Consider extracting data fetching logic to separate method and call it in `componentDidMount` and here.